src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/AppealReferenceNumberCcdHandlerHandlerTest.java

```diff
@@ -0,0 +1,129 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.handlers.presubmit;
```

Comment:
I see the file name as HandlerHandlerTest? Can this be changed to HandlerTest?

Reply:
Yes, good spot @rajeshthuraiyur - the existing name makes no sense.